build-system/tasks/dist.js

```diff
@@ -55,6 +60,37 @@ const {maybeUpdatePackages} = require('./update-packages');
 const {green, cyan} = colors;
 const argv = require('minimist')(process.argv.slice(2));
 
+const babel = require('@babel/core');
+const deglob = require('globs-to-files');
+
+function transferSrcsToTempDir() {
+  const tmp = `${tempy.directory()}/amphtml`;
+  process.env.AMP_TMP_DIR = tmp;
```

Comment:
I believe single pass stores the temp dir in an options field. Would it be possible to do so here as well, or if not, export the value as a const, and avoid the environment var altogether?

Reply:
fixed. i added the temp directory generation to sources.js and exported it